rdma_link_test.go

```diff
@@ -165,3 +166,42 @@ func TestRdmaLinkList(t *testing.T) {
 		t.Logf("%d: %s", link.Attrs.Index, link.Attrs.Name)
 	}
 }
+
+func TestRdmaLinkAddAndDel(t *testing.T) {
+	// related commit is https://github.com/torvalds/linux/commit/3856ec4b93c9463d36ee39098dde1fbbd29ec6dd.
+	minKernelRequired(t, 5, 1)
+	setupRdmaKModule(t, "rdma_rxe")
+
+	checkPresence := func(name string, exist bool) {
+		links, err := RdmaLinkList()
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		found := false
+		for _, link := range links {
+			if link.Attrs.Name == name {
+				found = true
+				break
+			}
+		}
+
+		if found != exist {
+			t.Fatalf("expected rdma link %s presence=%v, but got presence=%v", name, exist, found)
+		}
+	}
+
+	linkName := strings.ToLower(t.Name())
```

Comment:
> Any name is allowed. 

@paravmellanox Yes. So lower case is ok here, right?

Reply:
Yes, out of curiosity, why do you need to force lower case in this test?
Oftentimes, people learn the usage of methods by looking at UT.

Now as I do know nothing about rdma devices, I look at your UT and would think "Looks like device names have to be lower case, otherwise why would it do that in his own UT"

Reply:
Sounds good to me. I removed the `strings.ToLower`. Thanks for the suggestion. PTAL

Reply:
Thank you